constants.php

```diff
@@ -44,6 +44,7 @@
 
 defined('UPDATE_FILENAME') or define('UPDATE_FILENAME', DATA_PATH . '/update.php');
 defined('USERS_PATH') or define('USERS_PATH', DATA_PATH . '/users');
+defined('FAVICONS_PATH') or define('FAVICONS_PATH', DATA_PATH . '/favicons');
```

Comment:
Let's not add one global constant for all the folders we have ;-)
If you make that configurable, as is the case now, then you need to handle the case when it might be redefined to be elsewhere than `./data/favicons` for instance in our script to apply rights.
The rule so far is that only `./data/` and sub-folders should have write access (and `/tmp` or the system equivalent)

Reply:
I understand.
But ` DATA_PATH . '/favicons/'` is used 6 times in the code. I know the good practice that it is useful to reduce the string values that are often used and replace them with a constant.

Reply:
Yes, but then if possible, make that a class constant instead of a global constant